Examples/NoModules/Chapter 10/Ex10_03A.cpp

```diff
@@ -0,0 +1,28 @@
+// Using return type deduction with templates (decltype(auto) instead of auto)
+#include <iostream>
+#include <string>
+#include <vector>
+
+// Template for functions to return the larger of two values
+// Supports implicit converion of differently-typed arguments
+template <typename T1, typename T2>
+decltype(auto) larger(const T1& a, const T2& b)
```

Comment:
I don't have a decltype(auto) example in my text.
please verify.

Reply:
code here does compile and run.

Reply:
decltype(auto) example was added back, yes.